docs/src/main/tut/plots.md

```diff
@@ -170,45 +198,49 @@ Overlay(contours, initial)
 
 ## Faceting
 
-Imagine that after you made the visualization above, it turned out that you want to run similar simulations at different
-temperatures and additionally evaluate a different set of simulation parameters. This is a perfect candidate for
-faceting.
+Plotting WiFi locations is helpful, but what if we have even more data that we want to compare and plot using the same
+axis? EvilPlot's faceting lets us compare several plots at the same time. Let's grab data from
+<a href="https://data.boston.gov/dataset/snow-emergency-parking">snow parking</a>,
+<a href="https://data.boston.gov/dataset/tot-sprays">water playgrounds</a>,
+and <a href="https://data.boston.gov/dataset/polling-locations-20171">polling locations</a> and compare their respective
+contour plots. For the below example, load each of the CSVs into their
+own `Seq[Point]` and add each of them into a `Seq[Seq[Point]]` called `allData` in order so that WiFi is first, then
```

Comment:
"in order so that" -> "in order that"

Reply:
Oh... I understand now.  Instead add a comma in between: "in order, so that".

Reply:
Yep. Thanks!